proto/iotics/api/input.proto

```diff
@@ -72,6 +79,82 @@ message DeleteInputResponse {
 }
 // ---------------------------------------
 
+// CreateInputRequest describes a created input.
+message CreateInputRequest {
+  // CreateInputRequest payload.
+  message Payload {
+    // Input identifier string representation (simple string) (unique within the scope of a twin identifier's input set)
+    string inputId = 1;
```

Comment:
I think this should mirror the CreateFeedRequest - https://github.com/Iotic-Labs/api/blob/08a66595c0ef0eba49256adf5cea584613e74df7/proto/iotics/api/feed.proto#L61 - keep as `id`

Reply:
I know the rest are inputId but better to mirror Feeds for now then we can circle back and fix these things when we get a chance. 

Maybe @Elleppi has input? no pun intended

Reply:
Yeah, I actually realized this myself, it's not intuitive to use and inputId is a combined key of id, twinId, and hostId, so this naming is confusing. sorted this now.